packages/color-handle/README.md

```diff
@@ -1,45 +1,162 @@
+```
+
+**Internal Structure**: The component renders an inner div with the background color and an `<sp-color-loupe>` element that appears when the `open` property is true and the component is not disabled.
+
+### Options
+
+#### Color
+
+The `color` property sets the visual color displayed within the handle. This accepts any valid CSS color format. The default color is `rgba(255, 0, 0, 0.5)` (semi-transparent red).
+
+```html
+<div style="display: flex; gap: 16px; align-items: center; margin: 16px 0;">
+    <!-- Hex color -->
+    <div style="position: relative; height: 20px; margin: 20px;">
+        <sp-color-handle color="#ff0000"></sp-color-handle>
+    </div>
+
+    <!-- RGB format -->
+    <div style="position: relative; height: 20px; margin: 20px;">
+        <sp-color-handle color="rgb(255, 0, 0)"></sp-color-handle>
+    </div>
+
+    <!-- RGBA format with transparency -->
+    <div style="position: relative; height: 20px; margin: 20px;">
+        <sp-color-handle color="rgba(255, 0, 0, 0.5)"></sp-color-handle>
+    </div>
+
+    <!-- HSL format -->
+    <div style="position: relative; height: 20px; margin: 20px;">
+        <sp-color-handle color="hsl(0, 100%, 50%)"></sp-color-handle>
+    </div>
+
+    <!-- Named colors -->
+    <div style="position: relative; height: 20px; margin: 20px;">
+        <sp-color-handle color="red"></sp-color-handle>
+    </div>
+</div>
+```
+
+**Transparency Support**: When using transparent colors, the handle displays an opacity checkerboard pattern background to clearly show the transparency level.
+
+### States
+
+#### Standard
+
+The default state of the color handle, ready for interaction:
 
 ```html
 <sp-color-handle></sp-color-handle>
 ```
 
-## Disabled
+#### Disabled
+
+A disabled color handle shows that the control exists but is not available for interaction. This maintains layout continuity and communicates that the handle may become available later:
 
 ```html
 <sp-color-handle disabled></sp-color-handle>
 ```
 
-## Open
+#### Open
 
-When the `<sp-color-handle>` uses the `open` property, the `<sp-color-loupe>` component can be used above the handle to show the selected color that would otherwise be covered by a mouse, stylus, or finger on the down/touch state. This can be customized to appear only on finger-input, or always appear regardless of input type.
+When the `open` property is set, the `<sp-color-loupe>` component appears above the handle to show the selected color that would otherwise be covered by a mouse, stylus, or finger on the down/touch state. The loupe automatically appears for touch input (`pointerType === 'touch'`) and can be manually controlled for other input types.
 
 ```html
 <div style="height: 72px"></div>
 <sp-color-handle open></sp-color-handle>
 ```
+
+**Automatic Behavior**: The loupe automatically opens when touched and closes when the touch interaction ends. For mouse and stylus input, the loupe remains hidden by default unless explicitly set to `open="true"`.
```

Comment:
This shouldnt be needed, all details should be covered in the first paragraph, make updates there if you feel the wording needs updating